### PR TITLE
Allow to provide a clusterIP for the service

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
Fixes: #48

### CODE REVIEW

File: charts/dex/README.md [hunk 1]
```diff
@@ -140,6 +140,7 @@ ingress:
 | securityContext | object | `{}` | Container [security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container). See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1) for details. |
 | service.annotations | object | `{}` | Annotations to be added to the service. |
 | service.type | string | `"ClusterIP"` | Kubernetes [service type](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types). |
+| service.clusterIP | string | `""` | Internal cluster service IP (when applicable) |
```
Comment (sagikazarmark) [inline charts/dex/README.md:143]: Please update `README.md.gotmpl` instead and run `make docs`.

File: charts/dex/templates/service.yaml [hunk 2]
```diff
@@ -10,6 +10,9 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if .Values.service.clusterIP }}
+  clusterIP: {{ .Values.servoce.clusterIP }}
+  {{- end }}
```
Comment (sagikazarmark) [inline charts/dex/templates/service.yaml:15]:
```suggestion
  {{- with .Values.service.clusterIP }}
  clusterIP: {{ . }}
  {{- end }}
```

**Review by sagikazarmark — CHANGES_REQUESTED** [review]

File: charts/dex/README.md (hunk 1 above)
Comment (TeddyAndrieux) [reply charts/dex/README.md:143]: Right, didn't see this template, thank you

Comment (sagikazarmark) [reply charts/dex/README.md:143]: This still isn't right.

**Review by sagikazarmark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**